Iryna_Bondarenko/test/test.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var chai = require('chai');
+var expect = chai.expect;
+var chaihttp = require('chai-http');
+chai.use(chaihttp);
+
+require(__dirname + '/../server');
+
+describe('our server', function(){
+ it('should  get the time', function(done){
+   chai.request('localhost:3000')
+   .get('/time')
+   .end(function(err, res){
+		expect(err).to.eql(null);
```

Comment:
these are tabbed in too far - they just need to be indented once from the `.end()` on line 13